### PR TITLE
fix(typo): Fix hyphen usage

### PR DESCRIPTION
**Pull request by tibetiroka**
**Bugfix:** This PR fixes some erroneous hyphens/dashes/whatevers.

## Fix Details
Headshake is one word according to Merriam-Webster, though the wikipedia page has it as two words. The `head-shake` usage doesn't look common either way (though it is grammatically correct).

`Flyby`, `subtropical` and `stained glass` were also corrected to their standard forms.

## Testing Done
N/A

## Save File
N/A

### CODE REVIEW

**Comment by danaris** [comment]
Like with many noun phrases, "stained glass" is _correctly_ hyphenated when it is used as an adjective phrase instead (as in this case, "stained-glass patchwork").

**Comment by tibetiroka** [comment]
How sure are you about that? Searching for `"stained glass patchwork"` shows it as far, _far_ more common than `"stained-glass patchwork"`.

**Comment by danaris** [comment]
Extremely certain. However, it's also one of those rules of English that's not terribly well-understood (because it's not well-taught), so it doesn't surprise me that it's fading in casual usage.

**Comment by tibetiroka** [comment]
I looked it up again on [Merriam-Webster](https://www.merriam-webster.com/dictionary/stained%20glass). They give the example of

> a cathedral with beautiful stained glass windows

Why would that usage be any different than the one I fixed?

File: data/human/human missions.txt [hunk 1]
```diff
@@ -900,7 +900,7 @@ mission "WR Star 1"
 	passengers 3
 	cargo "sensors" 2
 	on offer
-		dialog `A group of <bunks> scientists approaches you and asks if you will be headed to the Rim any time soon. "We're doing research on Wolf-Rayet stars," they explain, "and we're hoping to find a captain who can do a fly-by of <waypoints>." Scientific research in the Deep is notoriously well-funded, so they will probably pay you quite well.`
+		dialog `A group of <bunks> scientists approaches you and asks if you will be headed to the Rim any time soon. "We're doing research on Wolf-Rayet stars," they explain, "and we're hoping to find a captain who can do a flyby of <waypoints>." Scientific research in the Deep is notoriously well-funded, so they will probably pay you quite well.`
```
Comment (ziproot) [inline data/human/human missions.txt:903]: Not needed. https://www.oed.com/dictionary/fly-by_n?tab=factsheet#3900446

**Review by ziproot — APPROVED** [review]
Everything else seems good.

File: data/human/human missions.txt (hunk 1 above)
Comment (ziproot) [reply data/human/human missions.txt:903]: Also, I don't think head-shake is grammatically correct. Head-shake is a verb; headshake is a noun.

Comment (tibetiroka) [reply data/human/human missions.txt:903]: > Not needed. https://www.oed.com/dictionary/fly-by_n?tab=factsheet#3900446

That might be the British way (though the [Cambridge dictionary would disagree](https://dictionary.cambridge.org/dictionary/english/flyby)), Merriam-Webster [only lists flyby](https://www.merriam-webster.com/dictionary/flyby) and actually redirects you to that page if you are looking for fly-by.

> Also, I don't think head-shake is grammatically correct. Head-shake is a verb; headshake is a noun.

So are you arguing for or against that fix?

Comment (ziproot) [reply data/human/human missions.txt:903]: > the [Cambridge dictionary would disagree](https://dictionary.cambridge.org/dictionary/english/flyby)

Here are some of the examples given:
>  These fly-bys provided the space probe with enough momentum to travel all the way out to the asteroid belt. 

>  Another lunar fly-by was performed later that month at a distance of 34,300 km to further improve the orbital

The Wikipedia article listed is the one on "Cassini-Huygens" which uses both fly-by and flyby. Hence, I approved the changes, but left a comment stating that one of them was unnecessary, as either spelling works.

2. I am arguing for the fix.

**Pull request merged**